source/deprecations/v2.x.html.md

```diff
@@ -753,3 +753,92 @@ component.appendTo('.my-component-wrapper');
 ```
 
 Note that both APIs are private, so no public API is been deprecated here.
+
+
+#### `{{render` helper
+
+##### until: 3.0.0
+##### id: ember-template-compiler.deprecate-render
+
+Using the `{{render` helper is deprecated in favor of using components.
+Please refactor to a component and invoke thusly:
```

Comment:
`and invoke thusly:` followed by another line with a `:` seems odd. Perhaps just "Please refactor uses of this helper to components".